### PR TITLE
add gzdoom package

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: pkgs/games/gzdoom/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  installPhase = ''
+    mkdir -p $out/bin
+    cp gzdoom $out/bin
+    mkdir -p $out/share
+    cp gzdoom.pk3 $out/share
+  '';
+
+  meta = {
+    homepage = https://github.com/coelckers/gzdoom;
+    description = "GZDoom adds an OpenGL renderer to the ZDoom source port.";
```
Comment (jagajaga) [inline pkgs/games/gzdoom/default.nix]: Don't add the name of the pkg in the description.

File: pkgs/games/gzdoom/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{stdenv, fetchFromGitHub, cmake, fmod, mesa, SDL2}:
+
+stdenv.mkDerivation {
+  name = "gzdoom-git";
```
Comment (nckx) [inline pkgs/games/gzdoom/default.nix]: SCM packages should have the date of the pinned commit as version number. In this case, that's `2015-05-07`.

File: pkgs/games/gzdoom/default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  installPhase = ''
+    mkdir -p $out/bin
+    cp gzdoom $out/bin
+    mkdir -p $out/share
+    cp gzdoom.pk3 $out/share
+  '';
+
+  meta = {
+    homepage = https://github.com/coelckers/gzdoom;
+    description = "Adds an OpenGL renderer to the ZDoom source port";
```
Comment (nckx) [inline pkgs/games/gzdoom/default.nix]: This sounds like it's an add-on/plug-in to the ZDoom package, which I presume it's not.

**Comment by lethalman** [comment]
Built locally, thanks.

**Pull request merged**